services/get-students.php

```diff
@@ -0,0 +1,89 @@
+<?php
+// File: /services/get_students.php
+
+// Set the content type to JSON
+header('Content-Type: application/json');
+
+// Shared secret for authorization
+$sharedSecret = "gradeplus";
+
+// Check authorization
+if ($authorize !== $sharedSecret) {
```

Comment:
I do not see any inputs being received from front-end. This variable would be undefined. `$authorize` should be `$_POST["authorize"]`